tensorboard/plugins/pr_curve/tf_pr_curve_dashboard/tf-pr-curve-dashboard.html

```diff
@@ -195,6 +196,12 @@ <h3>No precision–recall curve data was found.</h3>
           computed:
             '_makeCategories(_runToTagInfo, _selectedRuns, _tagFilter, _categoriesDomReady)',
         },
+        // Items show multiple runs, so exclude runs from category item keys for
+        // efficient template reuse.
+        _getCategoryItemKey: {
+          type: Function,
+          value: () => (item) => item.tag,
```

Comment:
I think this should just call https://github.com/tensorflow/tensorboard/blob/46e6f8c2e2d98ca2154af63d91d697137b1f966a/tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html#L294-L299.

Reply:
Ooops! `categoryKey` != `categoryItemKey`.

Reply:
Actually, I think _getCategoryKey is independent of _getCategoryItemKey, so the current PR is still correct.

After checking, the PR Curves dashboard still makes an extra network request.  I'll investigate that in a followup.